packages/components/htz-components/src/components/ArticleBody/ArticleBody.js

```diff
@@ -83,10 +83,14 @@ const Aside = ({ children, }) => (
 
 const buildComponent = (context, index, isLastItem) => {
   const uniqueId = context.elementType || context.inputTemplate || context.tag || null;
-  const Component = uniqueId === 'com.tm.Image' ? ArticleImage : getComponent(uniqueId);
+  const Component =
+    [ 'com.tm.Image', 'com.tm.BlogImage', ].indexOf(uniqueId) > -1
```

Comment:
```suggestion
    [ 'com.tm.Image', 'com.tm.BlogImage', ].includes(uniqueId)
```

I think the `include` method makes intent clearer